### PR TITLE
Use external repo for GoogleAppMeasurement SPM dependency

### PR DESCRIPTION
**Pull request by paulb777**
Confirmed in Analytics QS and CI.

Will rebase to master after #7315 merges

#no-changelog

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -96,6 +96,11 @@ let package = Package(
       url: "https://github.com/apple/swift-protobuf.git",
       from: "1.14.0"
     ),
+    .package(
+      name: "GoogleAppMeasurement",
+      url: "https://github.com/google/GoogleAppMeasurement.git",
+      from: "7.4.0"
```
Comment (ryanwilson) [inline Package.swift:102]: Do we want to do exact versioning here, or are they going to release in a different cadence?

Comment (paulb777) [reply Package.swift:102]: There is still a requirement to do exact matching, but the Analytics team will explore loosening in the future. cc: @allenkv @htcgh

Comment (ryanwilson) [reply Package.swift:102]: In that case we should use `.package(name: "GoogleAppMeasurement", url: "https://github.com/google/GoogleAppMeasurement.git", .exact("7.4.0")` (if that's the right syntax)

Comment (paulb777) [reply Package.swift]: Good catch. Thanks! Done.

**Review by ryanwilson — APPROVED** [review]

**Pull request merged**